pkg/transport/generic/cmd/options.go

```diff
@@ -33,6 +33,7 @@ type TransportOptions struct {
 	MaxSizeWrapped         int
 	MaxNumWrapped          int
 	WdsName                string
+	TransportName          string
```

Comment:
This is not the best choice of name. I think that "ITSName" would be better. It is parallel to "WdsName", and its meaning is clearer. Remember that KubeStellar also has the concept of pluggable transport, but this field is _not_ about that.